### ISSUE TITLE
External links to new screen

### ISSUE THREAD

**Issue by lvdhoorn** [nikans/text-linked#2]
It would be nice if external links open in a new window or if there is an options to make them open in a new window.

**Comment by omarmatijas** [comment]
I agree. It would be nice to add a "target" parameter or simple a boolean to do it.
For example, I have a "Pages" resource on CMS and want a link to each row to open it on front side.

**Comment by tamfelix** [comment]
great job, thank you so much!!
